### DISCUSSION TITLE
Hola la kata 10 no aparece en el Módulo 10

### DISCUSSION THREAD

**Discussion by aarqp** [LaunchX-InnovaccionVirtual/CursoIntroPython#329] [Q&A] [answered]
Hola buenas noches, ya me disponía a realizar la kata 10, sin embargo no carga el ejercicio.
![image](https://user-images.githubusercontent.com/94329640/154873596-d77d4908-0562-4be5-b285-d59050397482.png)

No sé si tengo que hacer algo para ver las instrucciones.

Espero que alguien me pueda ayudar pasando las instrucciones de la kata 10.

Saludos

**Comment by Samndamian** [comment, marked as answer]
Hola, buenas noches
La terminación es .md, es archivo word
Si estas visual es estudio puedes abrirla con una extensión que se llama **Office Viewer**
![image](https://user-images.githubusercontent.com/92234693/154874855-df63ed7a-1368-49bf-983c-8fc0f3ed8aa8.png)

**Comment by JuanSalvador73** [comment]
Lo primero es hacer los ejercicios descritos en el documento, tal y como están escritos. Esos los puedes hacer en un archivo py - de Python, claro - o mediante línea de comandos. También puedes usar la terminal de Visual Code, abajo, es lo que yo hice. Y salvas ese archivo, o puedes tomar screenshots si usas el CMD - línea de comando -. Yo personalmente usé los 2.
